src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    completed, title, user, id,
+  } = todo;
+
+  return (
+    <article data-id={id + 1} className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}>
```

Comment:
You are manipulating the `id` of the todo item by adding 1 to it. This may result in incorrect data-id attribute and can cause issues in future if relied upon. Consider using the original `id` value unless there is a specific reason for this manipulation.

Reply:
id is the largest id in the array + 1 (add data-id={todo.id} attribute to each .TodoInfo).